### PR TITLE
Injects Ubar in a custom container

### PR DESCRIPTION
**Pull request by rbelling**
Checks for an existing dom container element before creating a new node.
This is needed as UBAR might interfere with other dom elements, e.g. sticky navbars

https://hbcdigital.atlassian.net/browse/FECK-1944

### CODE REVIEW

File: js/ubar/dom.js [hunk 1]
```diff
@@ -79,29 +79,34 @@ function create (handlebars, when, request) {
...
-      ubarDiv.innerHTML = renderedHtml;
-      document.body.insertBefore(ubarDiv, document.body.firstChild);
-      self.banner = document.querySelectorAll('.' + self.MAIN_UBAR_CLASS)[0];
+      if (!ubarContainerElt) {
+        // No pre-existing UBAR container was found.
+        ubarContainerElt = document.createElement('div');
+        document.body.insertBefore(ubarContainerElt, document.body.firstChild);
+      }
+      ubarContainerElt.innerHTML = renderedHtml;
+      self.banner = document.querySelector('.' + self.MAIN_UBAR_CLASS);
```
Comment (kyle-dorman) [inline js/ubar/dom.js]: you can probably search from the `ubarContainerElt` instead of `document` here.

File: js/ubar/dom.js [hunk 2]
```diff
@@ -79,29 +79,34 @@ function create (handlebars, when, request) {
...
    * @param  {Object} templateSource The template to render
-   *
-   * @return  {Promise} Resolves after adding bannder to the dom, otherwise rejects.
+   * @return  {Promise} Resolves after adding banner to the dom, otherwise rejects.
    */
   UbarDom.prototype.renderBanner = function renderBanner (templateSource) {
     var
       self = this,
-      ubarDiv = document.createElement('div');
+      ubarContainerElt = document.querySelector('.' + self.UBAR_CONTAINER_CLASS);
```
Comment (kyle-dorman) [inline js/ubar/dom.js]: What is `Elt`?

Comment (rbelling) [inline js/ubar/dom.js]: Element, as we're dealing with a dom element (in a jquery world I would have declared it as `$ubarContainer`

**Comment by kyle-dorman** [comment]
👍  Feel free to merge

**Pull request merged**